src/constants.ts

```diff
@@ -136,6 +136,8 @@ export const PADDING_AUTORESIZE_HORIZONTAL = MIN_CELL_TEXT_MARGIN;
 export const FILTER_ICON_MARGIN = 2;
```

Comment:
I didn't checked the whole code, but does these two still need to be exported ?

Reply:
This should have been deleted, my bad